test/blocklib/core/fft/fft.hpp

```diff
@@ -179,160 +89,105 @@ struct fft : node<fft<T>> {
         ds.timestamp = 0;
         const std::size_t N{ magnitudeSpectrum.size() };
 
-        ds.axis_names   = { "time", "fft.real", "fft.imag", "magnitude", "phase" };
+        ds.axis_names   = { "frequency", "fft.real", "fft.imag", "magnitude", "phase" };
```

Comment:
as discussed: the names should be derived from the signal_name (see default tags) that is passed and modified through the blocks. To note .. the forwarded tag needs to modify the name, eg. "FFT('old name')" etc.